BMT:
  prep_time: 5
  exec_time: 5
  objects: 5
  ref_position: true
  tables: 2
  allowed_tables: [ "WS06", "WS11" ]
